.travis.yml

```diff
@@ -1,13 +1,33 @@
-language: objective-c
-osx_image: xcode11.4
-xcode_project: Example/Cloudinary.xcworkspace
-podfile: Example/Podfile
-xcode_scheme: travis_public_scheme
-xcode_sdk: iphonesimulator13.0
-install: pod install --project-directory=Example
+language:
+  - objective-c
+  
+osx_image:
+  - xcode12
+
+xcode_workspace:
+  - Example/Cloudinary.xcworkspace
+
+xcode_scheme:
+  - travis_public_scheme
+
+podfile:
+  - Example/Podfile
+
+install:
+  - pod install --project-directory=Example
+
+env:
+  - CLOUDINARY_URL=$CLOUDINARY_URL
+
+matrix:
+  include:
+    - xcode_destination: platform=iOS Simulator,OS=12.0,name=iPhone 8
+    - xcode_destination: platform=iOS Simulator,OS=13.0,name=iPhone 8
+    - xcode_destination: platform=iOS Simulator,OS=14.0,name=iPhone 8
 
 before_script: >
   export CLOUDINARY_URL=$(bash tools/get_test_cloud.sh);
   echo cloud_name: "$(echo $CLOUDINARY_URL | cut -d'@' -f2)"
 
-script: xcodebuild clean test -workspace Example/Cloudinary.xcworkspace -scheme travis_public_scheme -sdk iphonesimulator -destination 'platform=iOS Simulator,name=iPhone 8,OS=13.0' -CLOUDINARY_URL=$CLOUDINARY_URL | xcpretty && exit ${PIPESTATUS[0]}
+# script:
+#   - xcodebuild clean test -workspace Example/Cloudinary.xcworkspace -scheme travis_public_scheme -sdk iphonesimulator -destination "platform=iOS Simulator,OS=$OS,name=$NAME" -CLOUDINARY_URL=$CLOUDINARY_URL | xcpretty && exit ${PIPESTATUS[0]}
```

Comment:
Is this commented out since now we use the default travis/xcode test commands? If so we can remove this line altogether.